lib/builder/builder.js

```diff
@@ -146,6 +148,7 @@ module.exports = {
 	 * @param {Object} parameters Parameters
 	 * @param {Object} parameters.tree Dependency tree
 	 * @param {string} parameters.destPath Target path
+	 * @param {boolean} [parameters.clearDest=false] Decides whether project should clear the target path before build
```

Comment:
We discussed this internally and would favor renaming this feature to "**clean**Dest". 

Other tools seem to call this functionality "clean" rather than "clear" as well:  
- Maven: https://maven.apache.org/plugins/maven-clean-plugin/
- Grunt: https://github.com/gruntjs/grunt-contrib-clean
- make: https://www.gnu.org/software/make/manual/html_node/Cleanup.html